### PR TITLE
OEL-1421: Fix cosmetic changes.

### PR DESCRIPTION
**Pull request by Maxfire**
Fixed cosmetic, tests provided on showcase.

### CODE REVIEW

File: templates/content/node--oe-project--teaser.html.twig [hunk 1]
```diff
@@ -13,21 +13,26 @@
     label: _item,
   }]) %}
 {% endfor %}
+{% if content.oe_project_dates|field_value is not empty %}
+  {% set _content %}
+    <span class="text-muted text-nowrap me-4-5">{{ content.oe_project_dates }}</span>
+  {% endset %}
+{% endif %}
+{% if image is not empty %}
```
Comment (abel-santos-corral) [inline templates/content/node--oe-project--teaser.html.twig]: Consider if it's better to move this below as:
  'image': (image is not empty) ? image|merge({
    'path': image.src,
  }) : {},

**Review by abel-santos-corral — CHANGES_REQUESTED** [review]

**Review by abel-santos-corral — APPROVED** [review]
Approved

File: modules/oe_whitelabel_extra_project/oe_whitelabel_extra_project.module [hunk 2]
```diff
@@ -118,14 +118,24 @@ function _oe_whitelabel_extra_project_preprocess_featured_media(&$variables) {
...
   /** @var \Drupal\node\NodeInterface $node */
   $node = $variables['node'];
-
   $variables['inpage_navigation_links'] = [];
+  $variables['project_details_group'] = FALSE;
+
+  // Get field groups.
+  foreach ($variables['elements']['#fieldgroups'] as $group) {
+    foreach ($group->children as $field) {
+      if (isset($node->{$field}) && !empty($node->{$field}->getValue())) {
```
Comment (drishu) [inline modules/oe_whitelabel_extra_project/oe_whitelabel_extra_project.module]:
```suggestion
      if (isset($node->{$field}) && !$node->{$field}->isEmpty()) {
```

File: modules/oe_whitelabel_extra_project/oe_whitelabel_extra_project.module [hunk 3]
```diff
@@ -118,14 +118,24 @@ function _oe_whitelabel_extra_project_preprocess_featured_media(&$variables) {
 function _oe_whitelabel_extra_project_preprocess_inpage_nav(array &$variables): void {
   /** @var \Drupal\node\NodeInterface $node */
   $node = $variables['node'];
-
   $variables['inpage_navigation_links'] = [];
+  $variables['project_details_group'] = FALSE;
+
+  // Get field groups.
+  foreach ($variables['elements']['#fieldgroups'] as $group) {
```
Comment (drishu) [inline modules/oe_whitelabel_extra_project/oe_whitelabel_extra_project.module]: this is not completely correct, you should target only the groups that are part of the details group

File: templates/content/node--oe-project--teaser.html.twig [hunk 4]
```diff
@@ -13,21 +13,22 @@
     label: _item,
   }]) %}
 {% endfor %}
+{% if content.oe_project_dates|field_value is not empty %}
```
Comment (drishu) [inline templates/content/node--oe-project--teaser.html.twig:17]: remove the entire if

File: templates/content/node--oe-project--teaser.html.twig [hunk 5]
```diff
@@ -13,21 +13,22 @@
...
-</article>
+  <article{{attributes}}>
+    {{ pattern('card', {
+      variant: 'search',
+      title: _title,
+      text: content.oe_teaser|field_value,
+      image: (image is not empty) ? image|merge({
+        path: image.src,
+      }) : {},
+      content: _content,
```
Comment (drishu) [inline templates/content/node--oe-project--teaser.html.twig]: we have meta now for this, see the latest card changes
```suggestion
      meta: [
        content.oe_project_dates|field_value
      ],
```

**Review by drishu — CHANGES_REQUESTED** [review]

File: modules/oe_whitelabel_extra_project/oe_whitelabel_extra_project.module (hunk 3 above)
Comment (drishu) [reply modules/oe_whitelabel_extra_project/oe_whitelabel_extra_project.module]: I looked deeper into the issue, the problem is that the html element group prints the title even if the group is empty. So remove this solution, hide the title from the group, implement the old solution but add also striptags before trim. Then the old code works, but you have to add the "Project details" title in the template.

**Review by drishu — APPROVED** [review]

**Pull request merged**